tests/test_ramp_fitting.py

```diff
@@ -878,19 +980,34 @@ def test_zeroframe():
     # The third pixel in integration zero has good data
     # because the zeroframe has good data, so the ramp
     # is not fully saturated.
-    check = np.array([[[298.0626, np.nan, 652.01196]], [[18.62891, 18.62891, 18.62891]]])
+    if algo == DEFAULT_OLS:
+        check = np.array([[[298.0626, np.nan, 652.01196]], [[18.62891, 18.62891, 18.62891]]])
+    else:  # LIKELY
+        check = np.array([[[62.09636, np.nan, np.nan]], [[18.628912, 18.628912, 18.628912]]])
     np.testing.assert_allclose(cdata, check, tol, tol)
 
-    check = np.array([[[GOOD, DNU | SAT, GOOD]], [[GOOD, GOOD, GOOD]]])
-    np.testing.assert_allclose(cdq, check, tol, tol)
+    if algo == DEFAULT_OLS:
+        check = np.array([[[SAT, DNU | SAT, SAT]], [[GOOD, GOOD, GOOD]]])
+    else:  # LIKELY
+        check = np.array([[[SAT, DNU | SAT, DNU | SAT]], [[JUMP, JUMP, JUMP]]])
```

Comment:
Same as https://github.com/spacetelescope/stcal/pull/421/files#r2620906128 wrt JUMP. Also for one element, LIKELY has extra DNU. Is this correct?